react/SquareAppIcon/index.jsx

```diff
@@ -61,9 +61,16 @@ const useStyles = makeStyles(theme => ({
   }
 }))
 
-export const SquareAppIcon = ({ app, type, name, variant, IconContent }) => {
+export const SquareAppIcon = ({
+  name,
+  variant,
+  IconContent,
+  ...appIconProps
+}) => {
+  // "app" and "type" properties are deprecated in favor of appIconProps
```

Comment:
If you want to deprecate props, you should / can use https://github.com/cozy/cozy-ui/blob/67485ca3a6db28dd4ba64d2c91188c21b00034ce/react/helpers/migrateProps.jsx

Reply:
Actually, it's my mistake, these props are not deprecated anymore. I remove the comment.